### ISSUE TITLE
Does it handle form JSON Repeater?

### ISSUE THREAD

**Issue by chrisvidal** [mcharytoniuk/octobercms-headless#3]
I like the idea of this plugin.

Does it handle form JSON Repeater?
Thanks for your answer

**Comment by mcharytoniuk** [comment]
@chrisvidal yes it does, but if you want to use media in repeater you would need to use Media Finder widget: https://octobercms.com/docs/backend/forms#widget-mediafinder (you need to store image paths in JSON, because there are no CMS model `attachTo` relations that point to nested repeater fields). Other than that I think there are no problems. :)

**Comment by chrisvidal** [comment]
that's great to know, thanks

**Comment by chrisvidal** [comment]
just tested with json repeater and it works perfectly.

how about a relationship?
